feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/FilterChipWithDropdownMenu.kt

```diff
@@ -0,0 +1,145 @@
+package io.github.droidkaigi.confsched2023.sessions.component
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.material.icons.filled.Check
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FilterChip
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberUpdatedState
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+
+data class SearchFilterItemUiState<T>(
```

Comment:
How about using section package instead of component for this?
We don't use UiState for component.
https://github.com/DroidKaigi/conference-app-2023#section
And I would like to make this UiState's instance in ViewModel to avoid making a UiState in UI

Reply:
I think DropdownFilterChip and DropdownFilterChipUiState is better name for this 👀

Reply:
> I think DropdownFilterChip is better name for this 👀

FilterChipWithDropdownMenu rename DropdownFilterChip.
https://github.com/DroidKaigi/conference-app-2023/pull/784/commits/842adb92d9aa6a97b7f414751f68e3954f8fad89

> How about using section package instead of component for this?
We don't use UiState for component.

After much thought, I felt that `SearchFilterUiState` was a better fit, so I changed `SearchFilter` to section package.

I think the dependencies are as follows:

```
SearchScreen → SearchFilter → DropdownFilterChip
```

https://github.com/DroidKaigi/conference-app-2023/pull/784/commits/f06d51d20e09776fa00885875564c22c68cdc13d

Reply:
Looks good